sdk/core/core-http-compat/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 1.0.0 (2022-02-18)
```

Comment:
any reason to release this early in the sprint cycle?

Reply:
The plan is:

1. Release this package
2. Make the corresponding code changes in the generator to use this package. 
3. Release a new version of the generator
4. So, we could use the new version of the generator to migrate other (6 packages that we discussed) packages and also the communication packages.